molgenis-data-validation/src/main/java/org/molgenis/data/validation/ExpressionValidator.java

```diff
@@ -57,10 +58,12 @@ boolean resolveBooleanExpression(String expression, Entity entity)
 			return Collections.emptyList();
 		}
 
+		Bindings bindings = jsMagmaScriptEvaluator.createBindings(entity);
+
 		List<Boolean> validationResults = new ArrayList<>(expressions.size());
 		for (String expression : expressions)
 		{
-			Object value = jsMagmaScriptEvaluator.eval(expression, entity);
+			Object value = jsMagmaScriptEvaluator.eval(bindings, expression);
```

Comment:
According to the task for which this code was written evalList should be used instead of eval since this was measured to be a major performance bottleneck. Could you elaborate why this change wasn't made?

Reply:
I've switched to evalList method to hide the details.

Reply:
Could you add unit test(s) for ExpressionValidator.resolveBooleanExpression?